spec/linter-elixirc-spec.js

```diff
@@ -1,16 +1,115 @@
 'use babel';
 
 import { join } from 'path';
+import { remove } from 'fs-extra';
 
 const validPath = join(__dirname, 'fixtures', 'valid.ex');
-const warningPath = join(__dirname, 'fixtures', 'proj', 'mix.exs');
+const warningPath = join(__dirname, 'fixtures', 'proj', 'lib', 'proj.ex');
 const errorMode1Path = join(__dirname, 'fixtures', 'error-mode1.ex');
 const errorMode2Path = join(__dirname, 'fixtures', 'error-mode2.ex');
+const exsFilePath = join(__dirname, 'fixtures', 'script.exs');
+
+const mixBuildDirectory = join(__dirname, 'fixtures', 'proj', '_build');
+remove(mixBuildDirectory);
 
 describe('The elixirc provider for Linter', () => {
-  const lint = require('../lib/init.js').provideLinter().lint;
+  describe('when using the standard configuration', () => {
+    let lint;
+
+    beforeEach(() => {
+      atom.config.set('linter-elixirc.forceElixirc', false);
```

Comment:
I meant explicit in terms of showing what I was testing, rather than relying on the default. Also, _If_ the default changed in the future, the tests remain valid.